src/main/java/io/vertx/core/loadbalancing/LoadBalancer.java

```diff
@@ -26,6 +27,7 @@
  *
  * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
  */
+@FunctionalInterface
```

Comment:
I don't think this is what we want. Declaring functionnal interface has impact on the API and its evolution, it's usually good when it is a user API. This is not a user API (user shall not implement it).

Reply:
I think I accepted the IDE's suggestion by mistake, will remove, thanks.